core/templates/dev/head/pages/signup/signup.html

```diff
@@ -15,10 +15,6 @@
 
 {% block header_js %}
```

Comment:
I think you can remove this block entirely?

Reply:
If we remove this block then
https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/header_js_libs.html
these header files will also removed from the signup.html template

Reply:
@seanlip ?

Reply:
Ah, my bad, I thought it would use a default somehow. OK, lgtm!

(And thanks for the ping, sorry I missed this comment.)